internal/indexer/definitions/animebytes.yaml

```diff
@@ -82,5 +82,6 @@ irc:
           - uploader
 
     match:
+      infourl: "/torrents.php?id={{ .torrentId }}"
```

Comment:
This is wrong...
For non music torrents it is: https://animebytes.tv/torrents.php?id=00000&torrentid=0000000
For music torrents it is: https://animebytes.tv/torrents2.php?id=00000&torrentid=0000000
Where `id` is the ID for the torrent group and `torrentid` is for the torrent itself..

`torrentid` alone won't do it here unfortunatly..